### PR TITLE
Updated self managed on premises

### PR DESCRIPTION
**Pull request by bmckercher123**
Rewrite and new section for Get started rework: 

- Can merge at any time
- Reviewed by Spike
- Assign to Casey for final review
- Part of Alex reorg work: Alex provided inputs on scope of doc

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit af3d709ce049d5e321ed3c7c2ffb37a9e9170266

https://deploy-preview-3144--calico.netlify.com

File: getting-started/kubernetes/self-managed-onprem/onpremises.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+1. Customize the manifest as necessary. 
+1. Apply the manifest using the following command.
+
+   ```bash
+   kubectl apply -f calico.yaml
+   ```
+
+##### Install Calico with Kubernetes API datastore, more than 50 nodes
+
+1. Download the {{site.prodname}} Calico manifest (ConfigMap) for the Kubernetes API datastore.
```
Comment (lmm) [inline getting-started/kubernetes/self-managed-onprem/onpremises.md]:
```suggestion
1. Download the {{site.prodname}} manifest (ConfigMap) for the Kubernetes API datastore.
```

File: getting-started/kubernetes/self-managed-onprem/onpremises.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+>**Note**: The option, **Kubernetes API datastore, more than 50 nodes** provides scaling using {{site.prodname}} [Typha daemon](https://github.com/projectcalico/typha). Typha is not included for etcd because etcd v3 already handles many clients so using Typha is redundant and not recommended.
+{: .alert .alert-info}
+
+- [Install Calico with Kubernetes API datastore, 50 nodes or less](#install-calico-with-kubernetes-api-datastore-50-nodes-or-less)
+- [Install Calico with Kubernetes API datastore, more than 50 nodes](#install-calico-with-kubernetes-api-datastore-more-than-50-nodes)
+- [Install Calico with etcd datastore](#install-calico-with-etcd-datastore)
+
+##### Install Calico with Kubernetes API datastore, 50 nodes or less
+
+1. Download the {{site.prodname}} Calico manifest (calico-config.yaml) for the Kubernetes API datastore.
```
Comment (lmm) [inline getting-started/kubernetes/self-managed-onprem/onpremises.md]:
```suggestion
1. Download the {{site.prodname}} manifest for the Kubernetes API datastore.
```

File: getting-started/kubernetes/self-managed-onprem/onpremises.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+1. Customize the manifest if desired.
+1. Apply the manifest.
+
+   ```bash
+   kubectl apply -f calico.yaml
+   ```
+   
+##### Install Calico with etcd datastore
+
+1. Download the {{site.prodname}} Calico manifest (ConfigMap) for etcd.
```
Comment (lmm) [inline getting-started/kubernetes/self-managed-onprem/onpremises.md]:
```suggestion
1. Download the {{site.prodname}} manifest for etcd.
```

File: getting-started/kubernetes/self-managed-onprem/onpremises.md (hunk 1 above)
Comment (bmckercher123) [reply getting-started/kubernetes/self-managed-onprem/onpremises.md]: fixed

File: getting-started/kubernetes/self-managed-onprem/onpremises.md (hunk 3 above)
Comment (bmckercher123) [reply getting-started/kubernetes/self-managed-onprem/onpremises.md]: fixed

File: getting-started/kubernetes/self-managed-onprem/onpremises.md (hunk 2 above)
Comment (bmckercher123) [reply getting-started/kubernetes/self-managed-onprem/onpremises.md]: fixed

**Pull request merged**